pipelineutilities/build/lib/pipelineutilities/expand_getty_aat_terms.py

```diff
@@ -83,21 +83,23 @@ def _get_each_broader_term(xml: ElementTree, xpath: str) -> list:
 
 def _parse_parent_string_into_broader_terms(parent_string: str) -> list:
     """ Given the string of the parent hierarchy, return list of broader terms """
+    break_on_terms = ["styles, periods, and cultures by region"]
     broader_terms = []
-    parent_list = parent_string.split(",")
+    parent_list = parent_string.split("],")
```

Comment:
It seems you are breaking up a string that is structured with an established convention. I'm assuming this change is to avoid the inevitable comma in the middle of a term. This is probably susceptible to the same issue, albeit much less likely with the string "],"

I wonder if a regex solution might work better for this? You could match on both the term and the number and retrieve the value from each group. Something like `(.+?)\[(\d+?)\],?\s*`

Reply:
Good suggestion.  Thanks.